managed/utils/envvars/parser_test.go

```diff
@@ -145,57 +129,6 @@ func TestEnvVarValidator(t *testing.T) {
 		assert.Nil(t, gotWarns)
 	})
 
-	t.Run("PERCONA_TEST_PLATFORM_ADDRESS env vars with warnings", func(t *testing.T) {
-		t.Parallel()
-
-		envs := []string{
-			"PERCONA_TEST_PLATFORM_ADDRESS=https://host:333",
-		}
-		expectedEnvVars := &models.ChangeSettingsParams{}
-		expectedWarns := []string{
-			`environment variable "PERCONA_TEST_PLATFORM_ADDRESS" IS NOT SUPPORTED and WILL BE REMOVED IN THE FUTURE`,
-		}
-
-		gotEnvVars, gotErrs, gotWarns := ParseEnvVars(envs)
-		assert.Nil(t, gotErrs)
-		assert.Equal(t, expectedEnvVars, gotEnvVars)
-		assert.Equal(t, expectedWarns, gotWarns)
-	})
-
-	t.Run("PERCONA_TEST_CHECKS_PUBLIC_KEY env vars with warnings", func(t *testing.T) {
-		t.Parallel()
-
-		envs := []string{
-			"PERCONA_TEST_CHECKS_PUBLIC_KEY=some key",
-		}
-		expectedEnvVars := &models.ChangeSettingsParams{}
-		expectedWarns := []string{
-			`environment variable "PERCONA_TEST_CHECKS_PUBLIC_KEY" is removed and replaced by "PERCONA_TEST_PLATFORM_PUBLIC_KEY"`,
-		}
-
-		gotEnvVars, gotErrs, gotWarns := ParseEnvVars(envs)
-		assert.Nil(t, gotErrs)
-		assert.Equal(t, expectedEnvVars, gotEnvVars)
-		assert.Equal(t, expectedWarns, gotWarns)
-	})
-
-	t.Run("SAAS env vars with errors", func(t *testing.T) {
-		t.Parallel()
-
-		for _, k := range []string{
-			"PERCONA_TEST_AUTH_HOST",
-			"PERCONA_TEST_CHECKS_HOST",
-			"PERCONA_TEST_TELEMETRY_HOST",
-			"PERCONA_TEST_SAAS_HOST",
-		} {
-			expected := fmt.Sprintf(`environment variable %q is removed and replaced by "PERCONA_TEST_PLATFORM_ADDRESS"`, k)
-			envs := []string{k + "=host:333"}
-			_, gotErrs, gotWarns := ParseEnvVars(envs)
-			assert.Equal(t, []string{expected}, gotWarns)
-			assert.Nil(t, gotErrs)
-		}
-	})
```

Comment:
can we rework these 3 tests into one where we show warning about `PERCONA_` prefix being renamed?